pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1509,7 +1526,11 @@ def add_layer(
         name : str
             Layer name.
         layer_type : str, optional
-            Layer type. Options are ``"S"``, ``"D"``, and ``"G"``. The default is ``"S"``.
+            Layer type. Options are
+            - ``"S"`` for "signal" layer
+            - ``"D"`` for "dielectric" layer
+            - ``"G"`` for "ground" layer
+            The default is ``"S"``.
```

Comment:
```suggestion

            - ``"D"`` for "dielectric" layer
            - ``"G"`` for "ground" layer
            - ``"S"`` for "signal" layer

```
A blank line but also follow the last item in the list.